### DISCUSSION TITLE
Welcome to TinyTroupe Discussions!

### DISCUSSION THREAD

**Discussion by paulosalem** [microsoft/TinyTroupe#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas, references and findings.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, please comment below with an introduction of yourself and tell us about why you are interested TinyTroupe.

**Comment by sykespro** [comment]
This project looks promising and exciting!

**Comment by Ash112** [comment]
Would be good if we can have some examples of using models from other providers other than open AI.

**Reply by carter48fan** [reply]
I agree! I'd love to build this using my Anthropic or Gemini API keys.

**Comment by Evan-IT** [comment]
I haven t checked the detail code yet , but I have ran this project in my local env . It  s soooo cool  . But it confused me that what is the most differences between the TinyTroupe and the way we get the result from giving the prompt to llm .   Such as the advertisement example, there r four persons to discuss about which is the best ad , what if I give those ads to the openai to pick up the best one driectly?
